lib/rules/comma-style.js

```diff
@@ -82,21 +82,50 @@ module.exports = {
                     !astUtils.isTokenOnSameLine(previousItemToken, commaToken)) {
 
                 // lone comma
-                context.report(reportItem, {
-                    line: commaToken.loc.end.line,
-                    column: commaToken.loc.start.column
-                }, "Bad line breaking before and after ','.");
+                context.report({
+                    node: reportItem,
+                    loc: {
+                        line: commaToken.loc.end.line,
+                        column: commaToken.loc.start.column
+                    },
+                    message: "Bad line breaking before and after ','.",
+                    fix(fixer) {
+                        return fixer.removeRange([
+                            previousItemToken.range[1],
+                            commaToken.range[0]
+                        ]);
```

Comment:
This fixing seems to be for `style === "last"`.
If `style === "first"`, probably it should be `fixer.removeRange([commaToken.range[1], currentItemToken.range[0]])`.